### ISSUE TITLE
Finding structural "homologs"

### ISSUE THREAD

**Issue by deboramarks** [debbiemarkslab/EVcouplings#87]
We should search with HHPRED for all queries independent of alignment depth of search.
Just as an alert rather than to use and not to be put in web server.
( Current status with alignment search is under review from Thomas)

**Comment by thomashopf** [comment]
Yep would be great to have in to have HMM-HMM as well as HMM-sequence using hmmsearch. Now is the time since it is closely related to the current hmmsearch pull request (#86). 

What complicates this is that we would need a specific PDB HMM database as input - the compare submodule very consciously searches the corresponding uniprot sequences, *not* the PDB atom record or seqres sequences (which I think the presupplied pdb70 database contains). First thing to investigate would be if it is easy to create such a database.

**Comment by aggreen** [comment]
In my discussion with Debbie about this, we went back and forth on whether the HMMsearch output should be used in the comparison stage, or whether this should just be something that pops up as an alert for users on the webserver - letting them know that their protein may have a structure. 

We don't want to add functionality to the webserver that is not implemented in the pipeline, so we will have to implement this in the pipeline first.
